springwolf-core/src/main/java/io/github/stavshamir/springwolf/configuration/DefaultAsyncApiDocketService.java

```diff
@@ -73,7 +56,7 @@ private AsyncApiDocket parseApplicationConfigProperties() {
             builder.defaultContentType(configProperties.getDocket().getDefaultContentType());
         }
 
-        return builder.build();
+        docket = builder.build();
```

Comment:
NBC: I think it's more transparent to `return builder.build()` and then

```
    public AsyncApiDocket getAsyncApiDocket() {
        if (docket == null) {
            docket = createDocket();
        }
        return docket;
    }
``` 
In this way the check and initialization are in the same place, avoiding "hidden" side-effects